licensing/README.md

```diff
@@ -1,3 +1,66 @@
 # Licensing
 
-Placeholder for Licensing subproject
\ No newline at end of file
+This documents the Licensing subproject under SIG Release.
+The subproject is responsible for analyzing, reporting and remediating
+licensing concerns within the Kubernetes project orgs.
+
+## Licenses for Dependencies
+
+Dependencies that are licensed under Apache-2.0 do not require further license
+review or approval, since they are under the same license as Kubernetes itself.
+
+The IP policy in the [CNCF Charter], section 11, allows the CNCF Governing Board
+to review and approve other non-Apache-2.0 licenses on an exception basis.
+
+To streamline this process, licenses for some components are whitelisted as per
+the CNCF Whitelist Policy (adopted by the CNCF Governing Board on 2018-05-01).
+This whitelisting policy is described below.
+
+Components that are not under Apache-2.0, and that do not satisfy the Whitelist
+Policy, remain subject to review and exception approval by the Governing Board.
+
+In case of questions or concerns regarding the whitelist policy,
+please create an issue or send an email to the [SIG Release] mailing list.
```

Comment:
I dropped the SIG Architecture mailing list from here because this is primarily a SIG Release subproject.

Depending on the question/concern, we can include SIG Arch in the email thread if needed.